game-headed/src/main/java/org/triplea/game/client/ui/swing/laf/DefaultSubstanceLookAndFeelManager.java

```diff
@@ -16,14 +19,16 @@ public String getDefaultLookAndFeelClassName() {
   }
 
   @Override
-  public Collection<String> getInstalledLookAndFeelClassNames() {
+  public Collection<UIManager.LookAndFeelInfo> getInstalledLookAndFeels() {
     return SubstanceCortex.GlobalScope.getAllSkins().values().stream()
-        .map(skinInfo -> getLookAndFeelClassNameForSkinClassName(skinInfo.getClassName()))
+        .map(DefaultSubstanceLookAndFeelManager::newLookAndFeelInfoForSkin)
         .collect(Collectors.toList());
   }
 
-  private static String getLookAndFeelClassNameForSkinClassName(final String skinClassName) {
-    return skinClassName.replaceFirst("(?<=\\.)(\\w+)Skin$", "Substance$1LookAndFeel");
+  private static UIManager.LookAndFeelInfo newLookAndFeelInfoForSkin(final SkinInfo skinInfo) {
+    return new UIManager.LookAndFeelInfo(
+        "Substance " + skinInfo.getDisplayName(),
```

Comment:
I added the "Substance " prefix here to make the Substance L&Fs obvious, as they were before.  If that's not desirable, we can just remove it.  Or maybe a suffix (e.g. " (Substance)") would be less intrusive?